### PR TITLE
TFA Fix for the-Inconsistent Object not generated on the cluster.

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Jira Ticket - https://issues.redhat.com/browse/RHCEPHQE-14650

Report portal error - 
          https://reportportal-ocs3.apps.ocp-c1.prod.psi.redhat.com/ui/#rhcs/launches/all/2928/299322/299357/log
          https://reportportal-ocs3.apps.ocp-c1.prod.psi.redhat.com/ui/#rhcs/launches/all/2909/296608/296643/log

Fix:
  To fix the issue, I took ten objects and put them into a list, Carrying out further tests if any one object turns into an inconsistent object. Failure of the test case occurs if the script is unable to convert even one out of ten objects.

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
Reef Log- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-EXVU4R/

File: tests/rados/test_inconsistency_osd_epoch.py [hunk 1]
```diff
@@ -50,27 +50,39 @@ def run(ceph_cluster, **kw):
         obj_list = rados_obj.get_object_list(pool_name)
         log.debug(f"All the objects added onto the pool are : {obj_list}")
 
-        oname = random.choice(obj_list)
+        oname_list = random.choices(obj_list, k=10)
```
Comment (pdhiran) [inline tests/rados/test_inconsistency_osd_epoch.py:53]: Changes here look good. Can we add the same workflow to other tests where we expect inconsistent to be present? like : `tests/rados/test_osd_inconsistency_pg.py`?

That can be as part of another PR as well if needed.

**Review by pdhiran — COMMENTED** [review]
Changes Looks good

File: tests/rados/test_inconsistency_osd_epoch.py (hunk 1 above)
Comment (SrinivasaBharath) [reply tests/rados/test_inconsistency_osd_epoch.py:53]: I will take in another PR.

**Review by pdhiran — APPROVED** [review]
Looks good
